### ISSUE TITLE
Filter out full games

### ISSUE THREAD

**Issue by marleighnorton** [neinteractiveliterature/intercode#2910]
A request from the facebook group to be able to filter out full games.

**Comment by marleighnorton** [comment]
I think this is a duplicate, but don't have the other issue at hand

**Comment by dkapell** [comment]
Yes, I found four duplicates and merged them back into this one.
